src/test/java/DukeTest.java

```diff
@@ -0,0 +1,2 @@
+public class DukeTest {
```

Comment:
This seems interesting, what does this class do? I have not considered setting up a similar testing framework.